src/vivarium_gates_nutrition_optimization_child/model_specifications/nutrition_optimization_child.yaml

```diff
@@ -123,29 +112,7 @@ configuration:
 
     stratification:
         default:
-            - "sex"
-            - "age_group"
+            - 'sex'
         child_stunting:
             include:
-                - 'sqlns_coverage'
-        birth:
-            exclude:
-                - "age_group"
-            include:
-                - "maternal_supplementation"
-                - "bmi_anemia"
-        mortality:
-            include:
-                - "wasting_state"
-        diarrheal_diseases:
-            include:
-                - "wasting_state"
-        measles:
-            include:
-                - "wasting_state"
-        lower_respiratory_infections:
-            include:
-                - "wasting_state"
-        malaria:
-            include:
-                - "wasting_state"
+                - 'sqlns_coverage'
```

Comment:
Should these be removed or just commented out (i.e. are they going to be added back in at some point, or are they gone for good)?